src/core/sio.h

```diff
@@ -162,29 +72,190 @@ class SIO {
         bool isChained() const { return (allocState & ~1) == 0x52; }
     };
 
-    void getMcdBlockInfo(int mcd, int block, McdBlock &info);
+    static const size_t s_sectorSize = 8 * 16;            // 80h bytes per sector/frame
```

Comment:
constexpr, not const. Also, please use `c_` as a prefix for constants.

Reply:
Noted